src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -17,20 +17,34 @@ public class SortCommand extends Command {
     public static final String COMMAND_WORD_ASCENDING = "asort";
     public static final String COMMAND_WORD_DESCENDING = "dsort";
 
-    public static final String MESSAGE_FAILURE = "Sort only works in inspect window!";
-    public static final String MESSAGE_SUCCESS = "Deliveries have been sorted by ";
+    public static final String MESSAGE_SUCCESS_MAIN = "Contacts have been sorted by ";
```

Comment:
Seems to be a lot of constants, but might be due to having 2 command words, not sure how this can be improved.

Reply:
Will leave for now since attributes specified in the strings are different, meaning two different strings will have to be specified regardless.